lib/incito-browser/incito.ts

```diff
@@ -757,6 +764,41 @@ export default class Incito extends MicroEvent<{
             this.el.setAttribute('lang', this.incito.locale);
         }
 
+        // Inject Powered by Incito logo
+        if (this.displayBranding) {
+            const breakpoints = {
+                sm: 420,
+                md: 834,
+                lg: 1200
+            };
+            const paddings = {
+                sm: 8,
+                md: 10,
+                lg: 12
+            };
+            const breakpoint =
+                Object.keys(breakpoints).find(
+                    (name) =>
+                        (this.incito.root_view?.layout_width || 1200) <=
+                        breakpoints[name]
+                ) || 'lg';
+            const padding = paddings[breakpoint];
+
+            this.incito.root_view?.child_views?.push({
```

Comment:
@Jevy - this position works regardless of whether there's a outro section or not? I know we have had this as a bug in Bumblebee.

Reply:
@klarstrup @jrlarano . I think you pointed it out here - [here](https://github.com/tjek/bumblebee/issues/4010#issuecomment-1628342337)  that we currently have a similar logic to the above in BB but only for catalogs in the App (using API-keys). But we can currently see a difference between a view in the app vs in vizsla :
 If I open this link from my phone browser `https://etilbudsavis.dk/XL-BYG/tilbudsaviser/Robotplaeneklippere~p:K7wQE4xn`  I do not see the `powered by incito` but if I open it in the e*app then I see it. Sometimes I see a quick glitch of it -> and then it disappears. 
 
Is there something we need to be aware of here to render it correctly everywhere?      
If what I'm trying to say here doesn't make sense. Let me know.